src/actors/objects/sentry.rs

```diff
@@ -170,7 +176,7 @@ pub(super) fn download_from_source(
     });
 
     let response = Retry::spawn(
-        ExponentialBackoff::from_millis(100).map(jitter).take(3),
+        ExponentialBackoff::from_millis(10).map(jitter).take(3),
```

Comment:
The retry intervals were way too large. This would retry a single http request for over a minute.

Since we do a catch-all retry around the HTTP request we will also retry blacklisted IPs. It is unwieldy to not retry on a particular error as it's wrapped in a bunch of untyped actix errors.